### ISSUE TITLE
User feedback: VoiceOver doesn't announce selection state of card in target card pile when the card is selected

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#40]
On the user's device, VoiceOver only announces the name of the card. On my iOS 18.3 iPad, VoiceOver announces the name, and then says "selected". I suspect on the user's device, the 500ms delay that the app makes prior to announcing the selection string is not sufficient, and VoiceOver's default announcement of the name of the card is still truncating the app's custom announcement.

Note: On Android, TalkBack announces the custom unselected string when the target card pile card is unselected, but on both my and the user's iOS device. VoiceOver only says "off". So presumably VoiceOver's default announcements is truncating the custom string on both our machines.

**Comment by gbarkerz** [comment]
Fixed in V2.0.
